backend/main/server/models.py

```diff
@@ -1,33 +1,27 @@
 from main.server import app, db, ma
 from marshmallow import fields
 
-
-class User(db.Model):
-    __tablename__ = 'users'
-
-    id = db.Column(db.Integer, primary_key=True)
-    username = db.Column(db.String(50), nullable=False, unique=True)
-    password = db.Column(db.String(200), nullable=False)
-
-    def __init__(self, username, password):
-        self.username = username
-        self.password = password
-
-
 class Gallery(db.Model):
     __tablename__ = 'GALLERY'
     artworkID = db.Column(db.Integer, primary_key=True, autoincrement=True)
     artworkLink = db.Column(db.String(2048), nullable=False)
     artistLink = db.Column(db.String(2048), nullable=True)
     username = db.Column(db.String(64), nullable=True)
     title = db.Column(db.String(64), nullable=True)
+    __table_args__ = (db.UniqueConstraint('artworkLink'),
+            )
 
     def __init__(self, artworkLink, username, title, artistLink):
         self.artworkLink = artworkLink
         self.artistLink = artistLink
         self.username = username
         self.title = title
 
+    def __init__(self, data):
+        self.artworkLink = data.artworkLink
+        self.artistLink = data.artistLink
+        self.username = data.username
+        self.title = data.title
```

Comment:
Wait, does overloading constructor work here? I'm under the impression that function overload doesn't work in python unless you use some special overload decorators/libs.